core/src/test/java/io/questdb/test/griffin/engine/functions/conditional/CaseFunctionFactoryTest.java

```diff
@@ -906,7 +906,7 @@ public void testFloatToVarcharCast() throws Exception {
                         "    end \n" +
                         "from tanc",
                 106,
-                "inconvertible types DOUBLE to VARCHAR"
+                "inconvertible types: DOUBLE -> VARCHAR [from=DOUBLE, to=VARCHAR]"
```

Comment:
The old error is shorter with all the information needed, the new error repeats the types twice.

Reply:
sure, just wanted everywhere to use the same "inconvertible" check. I wanted to investigate why we log this twice, but perhaps on a different PR. The changed code had ah-hoc logging rather than common.